build.sbt

```diff
@@ -1216,6 +1216,7 @@ def allProjects =
     lmCoursierDefinitions,
     lmCoursier,
     lmCoursierShaded,
+    lmCoursierShadedPublishing,
```

Comment:
Nice catch. If that one is included, we probably won't need `lmCoursierShaded` above that one.

Reply:
ok, fine. I'll try to adjust the aggregate later then.